index.html

```diff
@@ -37,7 +37,7 @@ <h3>Mc</h3>
             <a class="nav-link" href="#banner" id="portfolio-link">Protfolio</a>
             <a class="nav-link" href="#about" id="about-link">About</a>
             <a class="nav-link" href="#services" id="contact-link">Contact</a>
-            <a href=""><i class="fa fa-envelope-square"></i></a>
+            <a class="env d-none" href=""><i class="fa fa-envelope-square"></i></a>
           </div>
```

Comment:
Please use the same font family from the design which is `Inter` and the font color is also not similar as in the design.

Reply:
hello, Thank you for helping me make my project better. but the problem here is that whenever I use  Inter font family I got a linter error, that is why I am not using it. thank you

Reply:
This issue is well resolved! ✔️